src/zeit/content/cp/interfaces.py

```diff
@@ -395,6 +398,10 @@ class IReadArea(zeit.edit.interfaces.IReadContainer):
         default=u'date-last-published-semantic desc',
         required=True)
 
+    referenced_topicpage = zope.schema.TextLine(
+        title=_('Referenced Topicpage'),
+        required=False)
```

Comment:
Das ist "the simples thing" das erstmal überhaupt was tut. Für den dynamic folder /thema reicht das auch völlig, weil da das XML eh programmatisch erzeugt wird. ;)

Aber ja, das sollte im vivi-UI dann in nem zweiten Schritt ein Typeahead oder sowas kriegen, weil durchaus geplant ist, Themenseiten-Inhalte auch "einfach mal so" in irgendwelchen Autoflächen anzuzeigen.